packages/ra-core/src/auth/useAuthenticated.spec.tsx

```diff
@@ -40,27 +46,52 @@ describe('useAuthenticated', () => {
             checkError: () => Promise.reject('bad method'),
             getPermissions: () => Promise.reject('bad method'),
         };
+        const store = createAdminStore();
+        const dispatch = jest.spyOn(store, 'dispatch');
+
         const FooWrapper = props => (
-            <AuthContext.Provider value={authProvider}>
-                <Authenticated {...props}>
-                    <Foo />
-                </Authenticated>
-            </AuthContext.Provider>
+            <Provider store={store}>
+                <CoreAdminContext
+                    authProvider={authProvider}
+                    dataProvider={() => Promise.resolve()}
+                >
+                    <Authenticated {...props}>
+                        <Foo />
+                    </Authenticated>
+                </CoreAdminContext>
+            </Provider>
         );
-        const { rerender, dispatch } = renderWithRedux(<FooWrapper />);
+        const { rerender } = render(<FooWrapper />);
         rerender(<FooWrapper authParams={{ foo: 'bar' }} />);
         expect(authProvider.checkAuth).toBeCalledTimes(2);
         expect(authProvider.checkAuth.mock.calls[1][0]).toEqual({ foo: 'bar' });
         expect(dispatch).toHaveBeenCalledTimes(0);
     });
 
     it('should not block rendering by default', async () => {
-        const { dispatch, queryByText } = renderWithRedux(
-            <Authenticated>
-                <Foo />
-            </Authenticated>
+        const authProvider = {
+            login: () => Promise.reject('bad method'),
+            logout: () => Promise.reject('bad method'),
+            checkAuth: jest.fn().mockResolvedValue(''),
+            checkError: () => Promise.reject('bad method'),
+            getPermissions: () => Promise.reject('bad method'),
+        };
+        const store = createAdminStore();
+        const dispatch = jest.spyOn(store, 'dispatch');
+
+        render(
+            <Provider store={store}>
+                <CoreAdminContext
+                    authProvider={authProvider}
+                    dataProvider={() => Promise.resolve()}
+                >
+                    <Authenticated>
+                        <Foo />
+                    </Authenticated>
+                </CoreAdminContext>
+            </Provider>
         );
-        expect(queryByText('Foo')).toBeDefined();
+        expect(screen.queryByText('Foo')).toBeDefined();
```

Comment:
same as for the Authenticated component

Reply:
As discussed, I didn't change the original tests. Testing more things si out of scope for this PR